packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -77,6 +85,255 @@ export class CommonWell {
   // https://specification.commonwellalliance.org/services/rest-api-reference (8.6.1 Error)
   // Note that also sometimes these calls 404 when things aren't found and etc
 
+  //--------------------------------------------------------------------------------------------
+  // Org Management
+  //--------------------------------------------------------------------------------------------
+
+  /**
+   * Create an org.
+   * See: https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Organization-Management-API---Overview-and-Summary.aspx#post-a-new-organization
+   *
+   * @param meta          Metadata about the request.
+   * @param organization  The org to create.
+   * @returns
+   */
+  async createOrg(meta: RequestMetadata, organization: Organization): Promise<Organization> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.post(
+      `${CommonWell.MEMBER_ENDPOINT}/${this.oid}/org`,
+      organization,
+      {
+        headers,
+      }
+    );
+    return organizationSchema.parse(resp.data);
+  }
+
+  /**
+   * Update an org.
+   * See: https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Organization-Management-API---Overview-and-Summary.aspx#post-a-new-organization
```

Comment:
nit: https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Organization-Management-API---Overview-and-Summary.aspx#put-new-information-into-an-organization